OnsenUI/OnsenUI
# chore(angularjs): Move AngularJS bindings to own package

As per discussion in #2476, this commit moves the AngularJS bindings into a new package
called angularjs-onsenui. For now, the bindings continue to be packaged with the core,
but with an added warning to users that they should take the opportunity to upgrade to
the new package before 2.11.0, when the bindings will be removed from the core package.

There are a large number of changed files, but the majority are test files that now reference the new location of `angularjs-onsenui.js`, as well as the new `bindings/angular1/node_modules` folder.

To go with this PR, I have also created a PR to update the docs with some info on how to migrate, which is linked from the warning message mentioned above. Though the link won't work yet, I'll make sure the docs are up before we publish 2.10.4.

Here are the test steps I have done:
* Run the e2e AngularJS tests
* Build the bindings using `gulp angular-bindings` from the project root, which is how it is currently done. I confirmed this builds the bindings to the `bindings/angular1/dist` folder, and also copies them to `build/js`. So, though the process is different, the result is the same as before, for now.
* Run `npm pack` in `bindings/angular1` to confirm what would be in the package if it were published. It contains `dist/angularjs-onsenui.js`, `dist/angularjs-onsenui.min.js`, `CHANGELOG.md` and `package.json`.
* Successfully built on Circle CI (all tests are passing)
